clients/nexus-client/src/lib.rs

```diff
@@ -95,6 +95,7 @@ impl From<omicron_common::api::internal::nexus::VmmState> for types::VmmState {
     fn from(s: omicron_common::api::internal::nexus::VmmState) -> Self {
         use omicron_common::api::internal::nexus::VmmState as Input;
         match s {
+            Input::Creating => types::VmmState::Creating,
```

Comment:
Potentially, we could try to keep the `Creating` state out of the API entirely, and just have it be an internal-to-Nexus DB state. Then we'd need to map it to something else in the Nexus API, even though I don't think anything *should* be seeing a VMM that's in `Creating` --- we could maybe just always return `Starting` publicly? I don't really know if it's worth doing that...